.ai/plans/livewire-write-authorization-hardening.md

```
@@ -0,0 +1,72 @@
+# Plan: Authorize the remaining public Livewire write methods
+
+## Status
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)